### DISCUSSION TITLE
Use "old" media_player entities

### DISCUSSION THREAD

**Discussion by mrwhite31** [music-assistant/support#876] [Q&A]
Hi all,
I have some automations playing local media and TTS on media_players. I'd like to keep them that way, because I don't want to have the announcement sound, or queues etc. active in my alarm system for example.
It seems to be possible to still use them, but they are only playing sound when the new mass media_player entities are switched on. Is that on purpose?
I would need to call the media_player.turn_on service in my automations for them to work correctly. It would be nice if the "old" media_player entities would work the same way as without MA.
Thanks for your feedback.

Edit: this is valid for media_player that don't support turn_on, turn_off services like sonos.

**Comment by marcelveldt** [comment]
For players that do not support power commands natively (e.g. cast, sonos), the mute will be used.
So at power on, the volume will be unmuted and at power off the volume will be muted
If you use the mass player and related service calls all this is handled for you.


We have a special service for the announcements which can automatically turn up volume, turn "on" the player etc.
My suggestion would be to use that and not the underlying service directly because it saves you a lot of work.
The announcement sound is optional, queue is bypassed for announcements. It is invented for exact this use case of yours.
Take a look at the "mass.queue_command" service which can do exactly what you want.

If you really insist of doing it all yourself and bypassing MA, you could issue an unmute before playback.
Kind of strange btw that Sonos doesn't unmute itself when playback starts but that aside.

**Comment by OzGav** [comment]
Old v1 info
